src/main/resources/matlab-bamboo-plugin.properties

```diff
@@ -33,6 +33,13 @@ matlab.test.coverage.exists= HTML code coverage
 matlab.model.coverage.exists= HTML model coverage
 matlab.code.coverage.directory= Folder path
 
+#Test run customizations
+matlab.test.customizations=Customize test run
+matlab.test.strict=Strict
```

Comment:
@mw-kapilg do  check with @mw-hrastega if the UI strings are okay, if you haven't.

Reply:
Sure, thanks!